otterdog/eclipse-dataspace-dcp.jsonnet

```diff
@@ -21,10 +21,13 @@ orgs.newOrg('eclipse-dataspace-dcp') {
       has_discussions: true,
       private_vulnerability_reporting_enabled: true,
       web_commit_signoff_required: false,
+      workflows+: {
+              default_workflow_permissions: "write",
```

Comment:
I would strongly advise against doing that. Your workflow already requests specific permissions, and I would not go to the default of always having write tokens because then you dont care anymore and its a pain to migrate a bunch of existing workflows to read permissions by default.